util/gitutil/detectgit.go

```diff
@@ -302,6 +302,17 @@ func detectGitAuthor(ctx context.Context, dir string) (string, error) {
 	return strings.SplitN(outStr, "\n", 2)[0], nil
 }
 
+// GlobalEmail returns the user's currently configured (global) email address
+func GlobalEmail(ctx context.Context) (string, error) {
+	cmd := exec.CommandContext(ctx, "git", "config", "--global", "user.email")
```

Comment:
I would use `git config --get user.email` rather than global. If a user happens to use a non-global email for certain projects, it's better to get the email specific email.